avro/src/main/java/com/fasterxml/jackson/dataformat/avro/AvroGenerator.java

```diff
@@ -43,7 +43,12 @@
          * of this header is not 100% reliably auto-detectable (while header has distinct marker,
          * "raw" Avro content has no limitations and could theoretically have same pre-amble from data).
          */
-        AVRO_FILE_OUTPUT(false)
+        AVRO_FILE_OUTPUT(false),
+
+        /**
+         * Feature that includes to every object type default value "null" when no real default value is defined.
```

Comment:
This should explain that setting only affects Schema generation, and NOT on reading or writing.